### ISSUE TITLE
order signs

### ISSUE THREAD

**Issue by ashumz** [ashumz/citycamp#49]
it's significantly less expensive to go through Kinko's for signage. Do we need free standing signs or do we have some kind of stand to hold the signs? If we do a plastic sign from Kinko's, there is the option to add grommets for hanging the sign. 

Curious what you guys did in the past and what is the best choice? Our tshirt vendor, Adam, offers free standing double sided signs, but they are quite a bit more expensive. I'll attach images of our options below

**Comment by ashumz** [comment]
![screen shot 2015-04-21 at 2 46 48 pm](https://cloud.githubusercontent.com/assets/100216/7260106/c1f0576a-e835-11e4-89bb-0b4d13b79f53.png)
As low as $36 for 18x24 sign (+ $4 for grommets to hang sign)

more sign packages/options from fedex/kinkos: 
![screen shot 2015-04-21 at 2 56 30 pm](https://cloud.githubusercontent.com/assets/100216/7260186/7210b98c-e836-11e4-9595-2ec9b4fcca08.png)

**Comment by ashumz** [comment]
![sign prices](https://cloud.githubusercontent.com/assets/100216/7260126/f1245ac2-e835-11e4-85be-07e09429d7aa.jpg)

Pricing for stickers and signs through tshirt vendor

**Comment by ashumz** [comment]
5-6 ftx 3 ft sponsor banner

**Comment by jhibbets** [comment]
We usually borrow easles from folks. We should get grommets too.

**Comment by ChrisTheDBA** [comment]
We have an OfficeMax corp discount card.  We get some unreal discounts.  We'll see what they can do too for comparison.

**Comment by ashumz** [comment]
That would be great @christhedba! I'd just be going to kinko's and ordering them a few days ahead of time, so if you have a better deal through OfficeMax that would be ideal. I can pick up the order if needed, too, if you place the order.

**Comment by ChrisTheDBA** [comment]
I've uploaded last years signs for inspiration/guidance to google drive

**Comment by ChrisTheDBA** [comment]
If we have proofs I can run them by Office Max and get a price.

**Comment by ashumz** [comment]
Called Office Max on Falls of Neuse and I think this is standard: 

They can do up to 24x36 within a 1-2 day turnaround. That sounds like what we need. 

They can mount on foam board for that size. 

They can also do a banner of that size. Would be nice to have the banner a little larger, but it will be a lot less expensive through OfficeMax than Kinko's (their cheapest banner is $90) 

I'll ping Carsi about the files we are using for print. 

@ChrisTheDBA which OfficeMax is closest to you? I can call them to ask/make sure what file type we need to give them so you have the right files. If you're up for it, you could order the signs since you have the discount card? 

Let me know which location and I'll get the info and talk with Carsi. 

Thanks! 

Ashley

/cc @jhibbets

**Comment by ChrisTheDBA** [comment]
The one in Garner is closest.  (919) 661-8950.  I think you can upload the proofs and I can pick them up and pay then.

**Comment by ashumz** [comment]
@christhedba they can only do a max of 24" wide or tall at that store. Any other locations close enough by? I think actually they need to be paid for online first so I plan to call whichever location we choose to find out how to work that out.

**Comment by ChrisTheDBA** [comment]
Not really, just let me know.  Just don't send me to Wake Forest or Durham. :)

**Comment by ashumz** [comment]
Carsi said she'll be done with the art tomorrow night so we will either get this done by Friday or early next week for pick up. The best way to do it might be for me to create an OfficeMax log in, share the log in with you once I've loaded up the shopping cart, and  have you do the payment/pick up (or I can pick up after you do the payment part since I might be closer in Raleigh.) I'll keep you posted :) I need to make sure we can submit it through online and get it printed in the store without any delays from them sending it off for printing.

**Comment by ChrisTheDBA** [comment]
So 2 24x36 Welcome Signs, 2 24x36 Sponsor Signs and 3-4 Yard Signs(18x24?) to point people in the right directions

**Comment by ChrisTheDBA** [comment]
Falls of Neuse is good.  We can run it up on Friday nad pay for it.  Need to do it in person to apply the discount.

**Comment by dtraleigh** [comment]
If the yard signs have space on them, we can just "paste" arrows on there to make them more useful and it's flexible, reusable. I love signs so I say 4 yard signs. :)

**Comment by TaxLadyTracy** [comment]
I did speak to the print center at the Garner Office Max yesterday and he said they can print them and mount to foam board for the 24x36.

**Comment by ashumz** [comment]
Thanks @TaxLadyTracy -- they told me they couldn't do larger than 24" high or wide! Glad to hear they can.  JUst checked in with Carsi to find out what the status of the files are for print

**Comment by ashumz** [comment]
She is waiting on a higher res Code for Raleigh logo from @jhibbets and then we'll have the files we need

**Comment by TaxLadyTracy** [comment]
Hey Ashley I have picked up some yard/real estate signs and stakes from my office that we can use to mount signs on.  I will go measure the open area but some of them have the arrows pre-printed on them :) (my job came in handy for once)

**Comment by TaxLadyTracy** [comment]
I have a total of  8 signs available for use.  All are 24x18
5 of them arrows pre-printed and the space to cover with our message is 13x22
1 has an arrow and 9x22 area to cover
1 is 15x22 are to cover.  
All are double sided and have the stakes .

If this doesn't make sense let me know.

**Comment by ashumz** [comment]
@ChrisTheDBA now that we have the files from Carsi, it looks like this is what we will want to order: 
1 CityCampNC vertical logo sign to display in an area in front of the Commons which has just the large logo (Carsi included both a horizontal and vertical sign, but I think the vertical logo looks much better for the 24x36 size.) 
-We will use that in front of HQ Raleigh too I would imagine

1 24x36 Welcome Sign for when you first walk in (we originally said 2, but I think if we have the sign above in front of the Commons, 1 welcome sign when you walk in is fine.) 
-Can also use this when you first walk in at HQ

 2 24x36 Sponsor Signs for when you first walk in and in the main area
-We should bring at least 1 for in front of 'stage' area at HQ

3-4 Yard Signs (6-8 print of the CityCampNC vertical logo) 
(I think these should be sized to fit Tracy's signs, and we can just tape them to the signs maybe? So 4-5 that are 13x22. Since the signs are double sides, maybe that means we get 8 13x22 print outs of the CityCampNC vertical logo? You'd know better than me based on the sign design. Maybe @dtraleigh has ideas on this too) OfficeMax should hopefully be able to scale what we already have from Carsi's files...  If not, let me know and I can probably make that happen.

Let me know if you have questions and if you're able to take care of all of the above. Thanks! 

Ashley

**Comment by ashumz** [comment]
also see #204 - make sure we have at least 4 stands/easels for all of the large 24x36 signs

**Comment by ChrisTheDBA** [comment]
Main signs ordered.  Pick up Wednesday

**Comment by ChrisTheDBA** [comment]
Signs are here. Pics posted.
